client/src/components/ElementEditor/InlineEditForm.js

```diff
@@ -1,11 +1,45 @@
+/* global window */
 import React, { PureComponent, PropTypes } from 'react';
 import classnames from 'classnames';
 import FormBuilderLoader from 'containers/FormBuilderLoader/FormBuilderLoader';
 import { loadElementSchemaValue } from 'state/editor/loadElementSchemaValue';
+import i18n from 'i18n';
 
 class InlineEditForm extends PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.handleLoadingError = this.handleLoadingError.bind(this);
+
+    this.state = {
+      loadingError: null
```

Comment:
I'm curious about whether we can use one of the injected GraphQL props (loading or error) from the Apollo query rather than defining our own state for it: https://github.com/dnadesign/silverstripe-elemental/blob/master/client/src/state/editor/readBlocksForPageQuery.js#L46-L48

These would be available as props on the component that the query is bound to